2 commits, an initial translation of c00/30/04->14 and an extra adding sounds as per indication. planning to use dupe tool after this (#63)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/10/1/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//

> 我慢が出来ず押し入ってしまいました。我慢が出来なかったのはお手洗いではなく、性欲の方ですが。=I couldn't hold it any more, so I broke in. It wasn't the bathroom that I couldn't resist, but my sexual desire.

Reviewer: She's saying her had to rush to the bathroom due to her urges, but not her urge to go to the bathroom.  While not literal, this may be more accurate:

```suggestion
我慢が出来ず押し入ってしまいました。我慢が出来なかったのはお手洗いではなく、性欲の方ですが。=I couldn't hold it any longer, so I rushed in here. I don't need to go, I need to come.
```

Author: ^ this, now THIS is briliant!

> 精液も溜まっているのではないですか？　ついでに出してスッキリしましょう。精液専用の便器はこちらです。=I'm sure you have a lot of pent up semen. Let it out right away and clear your mind. Here is your semen toilet.

Reviewer: ```suggestion
精液も溜まっているのではないですか？　ついでに出してスッキリしましょう。精液専用の便器はこちらです。=Do you have a lot of semen in there? You should release it to clear your system. I'll be your cum toilet.
```
